src/edu/ucsb/cs56/w15/drawings/wpollek/SimpleGui1.java~

```diff
@@ -0,0 +1,27 @@
+package edu.ucsb.cs56.w15.drawings.wpollek;
+import javax.swing.*;
+
+/** SimpleGui1 comes from Head First Java 2nd Edition p. 355.
+      It illustrates a simple GUI with a Button that doesn't do anything yet.
+
+     @author Head First Java, 2nd Edition p. 355
```

Comment:
Sorry about that! I should have read through what I was committing better before sending it